rmd - readme - hw01.Rmd

```diff
@@ -0,0 +1,33 @@
+---
+title: "Bio-hw01"
+author: "Frank"
+date: "4/8/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+
+# My personal introduction
+
+## Brief Introduction
+
+My name is Feiyang **Frank** Yu. I am from MAPSS program, majored in political science and sociology. I am from the southern part of China. I have try to learn *Python* by myself two years ago, and this is the first time I learn something about R. It seems to me that the use of R is like using the functions in Python. It is efficient for data scientists, which is very helpful to me as well.
+
+![Photo of myself](/Users/macbook/Desktop/graduation.jpeg)
```

Comment:
This image will not render because it uses a finite filepath instead. Additionally, it doesn't appear that you've pushed it to github. We actually **DO NOT** recommend you upload images through github directly. That can create many problems in more complicated, collaborative projects so we try not to encourage the habit here. To include your image in your project repo, go to RStudio server where you wrote your README for this project and click the upload button in the bottom right hand corner by your file manager. There, you can upload a file to your project, and then commit and push up the image through RStudio directly instead of through GitHub. That will also allow you to use a relative filepath instead of one that is defined on your computer. My users folder is "Neeraj Sharma" so trying to find an image through a "JessicaLee" directory won't work!